nefertari/utils/__init__.py

```diff
@@ -3,3 +3,13 @@
 from nefertari.utils.utils import *
 
 _split = split_strip
+
+
+class SingletonMeta(type):
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(SingletonMeta, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
+
```

Comment:
Once this is merged I think you can remove the implementation in cerri_api/celery/utils.py